pvlib/spectrum/mismatch.py

```diff
@@ -358,21 +358,17 @@ def integrate(e):
 def spectral_factor_firstsolar(precipitable_water, airmass_absolute,
                                module_type=None, coefficients=None,
                                min_precipitable_water=0.1,
-                               max_precipitable_water=8):
+                               max_precipitable_water=8,
+                               min_airmass_absolute=1.0,
```

Comment:
```suggestion
                               min_airmass_absolute=0.58,
```
This value doesn't match the docstring. From the discussion it seems that there's agreement on using 0.58 as this was implemented in pvlib by the author.